components/board/numato/darwin_utils.go

```diff
@@ -0,0 +1,12 @@
+//go:build darwin
+
+package numato
+
+import "go.viam.com/utils/usb"
+
+// getSerialDevices returns all devices connected by USB on a mac.
```

Comment:
Comment doesn't really match code: I think this returns descriptions of devices, not devices themselves. and yet, I can get that from the function signature, so it doesn't necessarily need to be in the comment, too. but I still don't easily see what this function is for. I think the point is that we use it to find the productID of the numato board plugged in, and use that to figure out what capabilities the board has? That's the part that really belongs in a comment: supply the "why" that isn't easily gotten from the code (which supplies the "what").

Same suggestion in linux_utils.go, too.